### PR TITLE
Feature/multi region support

### PR DESCRIPTION
**Pull request by raksha1**
Updated plugin to support docker image push to multiple regions. This will ensure images are available across multiple regions with the same digest value. 
`region` property has been updated to `regions`. This could lead to a breaking change.
@72636c can you please have a look?

### CODE REVIEW

**Comment by 72636c** [comment]
Hi @raksha1, thanks for your contribution 🙇.

I'd prefer to avoid a breaking change if possible. Can we retain support for the existing `region` option or even merge this into that option? For example, Buildkite steps support both `command` and `commands`, and either can take a single string or an array.

File: plugin.yml [hunk 1]
```diff
@@ -28,11 +28,15 @@ configuration:
...
+      type: [array, string]
     tags:
       type: [array, string]
     target:
       type: string
     add-latest-tag:
       type: boolean
   required: ['ecr-name']
+  not:
+    required: ['region', 'regions']
```
Comment (72636c) [inline plugin.yml:42]: 👏

File: README.md [hunk 2]
```diff
@@ -221,10 +232,14 @@ steps:
...
   Account ID for ECR registry, defaults to output of `aws sts get-caller-identity` e.g. current account ID.
 
-- `region` (optional, string)
+- `region` (optional, array|string)
 
   Region the ECR registry is in, defaults to `$AWS_DEFAULT_REGION` and then `eu-west-1` if not set.
 
+- `regions` (optional, array|string)
+
+  Same as `region`. Both `region` and `regions` cannot be set. But they are optional.
```
Comment (72636c) [inline README.md]:
```suggestion
  Same as `region`. Both properties are optional, and they cannot be set at the same time.
```

**Review by 72636c — APPROVED** [review]
Awesome 🤩

**Comment by raksha1** [comment]
Thanks a lot for your feedback, @72636c! :) 
Makes more sense to support both `region` and `regions`

File: plugin.yml (hunk 1 above)
Comment (raksha1) [reply plugin.yml:42]: Thank you!

**Pull request merged**